locales/de.json

```diff
@@ -38,5 +38,11 @@
     "Next post": "Nächster Beitrag",
     "Previous post": "Vorheriger Beitrag",
     "Sign in": "Anmelden",
-    "Sign up": "Mitglied werden"
+    "Sign up": "Mitglied werden",
+    "This post is for paying subscribers only": "Dieser Beitrag ist nur für zahlende Mitglieder",
+    "This post is for subscribers only": "Dieser Beitrag ist nur für Mitglieder",
+    "This page is for paying subscribers only": "Diese Seite ist nur für zahlende Mitglieder",
+    "This page is for subscribers only": "Diese Seite ist nur für Mitglieder",
+    "Already have an account?": "Hast du schon einen Account?",
+    "Don't have an account yet?": "Du hast noch keinen Account?",
```

Comment:
Do they work for you? For me those strings don't translate. Unless they work somehow I'd keep them out for the moment.

Reply:
I did not get the chance to test these, sorry! I will remove them then.